### ISSUE TITLE
Doesn't list directories whose descendant files are all deeply located

### ISSUE THREAD

**Issue by hkurokawa** [pindexis/qfc#14]
When I ran `qfc` at a directory where the file structure is as following, I had no result.

```
[hiroshi ~/test-qfc]% tree
.
└── A
    └── B
        └── C
            └── empty.file

3 directories, 1 file
[hiroshi ~/test-qfc]% qfc
Path: 
Nothing found
```

Is this behavior as expected?  It seems if a directory does not have a regular file in its child or grand child directories, the directory itself is not listed up.

Some environment require a rather deep directory structure and I am wondering it could be a pain.  For example, in Go, source code is typically located under `~/go/src/<vc repository>/<organization>/<repository name>/` and I have to move to, say, `~/go/src/github.com` to list source files with `qfc` command.

Thanks in advance for your support.

**Comment by pindexis** [comment]
This has to be changed, empty dirs have to be displayed.

btw, this is not a git managed directory, right?

**Comment by hkurokawa** [comment]
Right.  I tried the same directory structure under a git repository and confirmed `qfc` could list up the empty directories in that case.

Thank you for your clarification.  I am looking forward to an enhancement on that.
